### PR TITLE
History of ARGs section

### PR DESCRIPTION
**Pull request by a-ignatieva**
I have restructured this section a bit to make clearer the different research directions involving ARGs (plus some other minor edits).

### CODE REVIEW

**Review by jeromekelleher — COMMENTED** [review]
Thanks for this @a-ignatieva - it is quite a big change to the current narrative though, particularly the split into separate named paragraphs.

Maybe we should be more explicit in the last para that the purpose is to try to explain the different meanings of the term and where they came from, rather than be real history up to the present day?

File: paper.tex [hunk 1]
```diff
@@ -173,102 +173,59 @@ \section*{Ancestral graphs: a brief history}
...
+
+Subsequent work on ARGs proceeded in broadly three main directions: focussing on (1) 
+exploring the mathematical properties of the coalescent with recombination and 
+related stochastic processes, (2) inferring evolutionary parameters under 
+(approximations to) this model, either with or without explicitly reconstructing the 
+genealogy of the sample, and (3) treating the ARG as a discrete graph, ignoring the 
+generating stochastic process, and studying its properties from a computational and 
+algorithmic perspective.
+
+\paragraph*{Properties of the coalescent with recombination and related models}
```
[the remark this answers is not in the document]
Comment (a-ignatieva) [reply paper.tex:216]: Happy to remove the naming! It was more for the sake of delineating the purposes of the paragraphs. Is the main concern that the section is too long? Just seemed hard to explain the different contexts within which ARGs are used without some discussion of the methods.

File: paper.tex [hunk 2]
```diff
@@ -284,10 +241,71 @@ \section*{Ancestral graphs: a brief history}
...
+fearnhead2001estimating}.
+These methods met with limited success
+because the state space of ARGs is overwhelmingly large, and
+lacks a simple geometry or neighbourhood structure for inference or
+sampling methods to  exploit.
+Several breakthroughs in this direction were achieved through
+formulating simplified but more tractable approximations to the full 
+model~\citep{mcvean2005approximating,marjoram2006fast,li2011inference,
+paul2011accurate,schiffels2014inferring}.
+The related problem of \emph{sampling} genealogies compatible with a given
```
[the remark this answers is not in the document]
Comment (a-ignatieva) [reply paper.tex:260]: Does the section need renaming? Maybe I've just misinterpreted the point of the section. I thought a brief discussion of the different streams of work on ARGs (which then requires some mention of the methods), with the final paragraph to point out that the same terminology is used to describe very different processes/objects.

[the remark this answers is not in the document]
Comment (a-ignatieva) [reply paper.tex:260]: (I didn't think I added a lot of extra "history")

**Comment by jeromekelleher** [comment]
How about we merge iterate@a-ignatieva - but lets remove the \paras, and maybe put in a quick summary of the purpose of each para as a comment?

File: paper.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeromekelleher) [reply paper.tex:260]: Maybe the section does need renaming all right...

**Comment by a-ignatieva** [comment]
Sounds good! Happy to revise this later.

**Pull request merged**